cmd/scaffold.go

```diff
@@ -207,7 +209,8 @@ func (fnb *FlowNodeBuilder) enqueueNetworkInit() {
 			fnb.Me.NodeID(),
 			fnb.Metrics.Network,
 			fnb.RootBlock.ID().String(),
-			p2p.DefaultPeerUpdateInterval,
```

Comment:
this was wrong - I should be using the variable and not the constant for `peerUpdateInterval`.